config/config.template.js

```diff
@@ -1,16 +1,18 @@
 module.exports = {
-    port: 8000,
-    //cdns: ['bootstrap', 'cdnjs', 'google', 'jsdelivr', 'jquery'],
-    cdns: ['bootstrap', 'cdnjs', 'google', 'jsdelivr'],
-    syncUrl: 'http://jsdelivrapi-sync.aws.af.cm/data/',
-    tasks: {
-        sync: {minute: 2}
-    },
-    maxcdn: {
-        alias: 'replace this',
-        key: 'replace this',
-        secret: 'replace this',
-        zoneId: 0 // replace with some zone id
-    },
-    github: '' // replace with github token
```

Comment:
In the future can you change indentation/styles in separate PRs makes the diff really hard to read